tests/e2e/config_secret.go

```diff
@@ -981,8 +981,8 @@ var _ = ginkgo.Describe("Config-Secret", func() {
 		10. Cleanup all objects created during the test
 	*/
 
-	ginkgo.It("[csi-config-secret-block][csi-config-secret-file] Add wrong datacenter and switch back "+
-		"to the correct datacenter in vsphere config secret file", func() {
+	ginkgo.It("Add wrong datacenter and switch back to the correct datacenter in vsphere "+
+		"config secret file", ginkgo.Label(p1, vsphereConfigSecret, block, file, vanilla), func() {
```

Comment:
Should we add stable label  to config secret tests as i have seen these test cases pass consistently

Reply:
I added it before, but later removed it
I wanted to monitor a few more config and topology runs for some more time to check the stability of the code.
Planning to add stable/flaky Labels in the next PR I will be sending for topology and other features

Reply:
okay